force-app/main/default/classes/ClassesObjectsMethods.cls

```diff
@@ -101,17 +104,14 @@ public with sharing class ClassesObjectsMethods {
      * @return The number of vowels in the string.
      */
     public static Integer countVowels(String str) {
-        // Initialize the result Integer
-
-        /*
-        for () {
-            // get individual characters from the string - hint use substring or split
-            // check if the character is a vowel
-            // if it is a vowel, increment the result Integer
+        Integer totalNumVowels = 0;
+        List<String> charactersSplit = str.split('');
+        for (String character : charactersSplit) {
+            if (character.containsAny('aeiou')) {
+                totalNumVowels += 1;
+            }
```

Comment:
I really like this approach; the other way is to check if a Set<String> or string of vowels contains the current char.

Reply:
ContainsAny is really great though, and a lot of people don't use it.